x-pack/test/api_integration/apis/management/rollup/rollup.js

```diff
@@ -24,7 +24,7 @@ export default function ({ getService }) {
     cleanUp,
   } = registerHelpers({ supertest, es });
 
-  describe('jobs', () => {
+  describe('Rollup jobs', () => {
```

Comment:
This was written like this on purpose as each "describe" creates a section.
Its parent is "Rollup". So we would read the test like this: Rollup -> jobs -> Index patterns -> <should ...>.